### ISSUE TITLE
Mutable types as default values

### ISSUE THREAD

**Issue by Ishrock** [Adyen/adyen-python-api-library#77]
**Python version**: 3+
**Library version**: 2.1.0
**Description**
Default values in functions should not be mutable types in python.  The value set will be maintained between function calls

This is well known behaviour in python.

```python
>>> def mutable_default_test(default={}):
...     print(default)
...     default["key"] = "value"
...
>>> mutable_default_test()
{}
>>> mutable_default_test()
{'key': 'value'}
>>> mutable_default_test({"something": "some value"})
{'something': 'some value'}
>>> mutable_default_test()
{'key': 'value'}
```

**Comment by Ishrock** [comment]
passing pull request submitted for fix: #78

**Comment by AlexandrosMor** [comment]
PR #78 Merged
